src/redux/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux'
```

Comment:
Not sure whether there is a unnecessity to create a file for this, I would personaly just initialize rootReducer  in the _store.js_ file and use it there, but if you think that this structure is better than okay :)

Reply:
Here's what I think: we have a separate folder for reducers and `rootReducer` is a reducer as well, so I moved it to the /reducers folder :)

Reply:
good, lets leave it there